task4/sorting.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	//"sort"
+	"time"
+)
+
+type Interface interface {
```

Comment:
It's not required to declare this interface here. 
Once struct `Person` will implement all func from ANY interface, it fits.
So, let's say `Swap` will be uncommented, `Person`will comply `sort.Interface` and `main.Interface`,
but func `sort.Sort` requires `sort.Interface`.